PARAMETERS_MAPPING/parameters_mapping.csv

```diff
@@ -138,6 +138,26 @@ AATAMS;sattag;dm;sal_vals;12;481;;
 AATAMS;sattag;dm;sal_corrected_vals;12;481;;
 AATAMS;sattag;dm;temp_dbar;23;445;;
 AATAMS;sattag;dm;sal_dbar;23;445;;
+AATAMS;sattag;qc_nrt;lat;4;440;Degrees North;
+AATAMS;sattag;qc_nrt;lon;5;440;Degrees East;
+AATAMS;sattag;qc_nrt;pressure;23;445;
+AATAMS;sattag;qc_nrt;temp_vals;13;429;;
+AATAMS;sattag;qc_nrt;sal_vals;12;481;;
+AATAMS;sattag;qc_nrt;sal_corrected_vals;12;481;;
+AATAMS;sattag;qc_nrt;cond_vals;2;;Millisiemens per centimetre;

```

Comment:
2 comments: 
first you should use the unit id, as  defined in the unit_view( I added the unit last week)
Second, the unit is the 6th field  but you put it in the 7th position

Reply:
Thanks @bpasquer , I forgot to update that one after you updated the view!... Now resolved